slurmstepd: info: Setting TMPDIR to /scratch/10927293. Previous errors about TMPDIR can be discarded
Slurm job started at 10/03/2025_12:13:32 PM

In transit.
Initing Ray (1 head node + 2 worker nodes + 2 simulation nodes) on nodes: 
	ilk-172 ilk-173 ilk-175 ilk-177 ilk-180
Head node: 172.30.204.28:6379


RAY_DEPLOY_TIME:     24.43241
RAY_INIT_TIME:       3.65657
Iter [0]
2025-03-10 12:14:02,682	INFO packaging.py:520 -- Creating a file package for local directory '/mnt/lustre/scratch/nlsas/home/ulc/cursos/curso341/dask_on_reisa_tests/E4/DASK_ON_REISA/reduction/P4-SN2-LS536-GS2147-I20-AN2-D2025-03-10_08-46-49'.
2025-03-10 12:14:02,708	INFO packaging.py:347 -- Pushing file package 'gcs://_ray_pkg_b2fedd141b826b81.zip' (0.48MiB) to Ray cluster...
2025-03-10 12:14:02,715	INFO packaging.py:360 -- Successfully pushed file package 'gcs://_ray_pkg_b2fedd141b826b81.zip'.
[2m[36m(iter_task pid=3077195, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077195, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077194, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077194, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077200, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077200, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077197, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077197, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077203, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077203, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077202, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077202, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077196, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077196, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077198, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077198, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077201, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077201, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077199, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077199, ip=10.120.204.29)[0m   warnings.warn(
Iter [2]
[2m[36m(iter_task pid=3077812, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077812, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077263, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077263, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077410, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077410, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077453, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077453, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077720, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077720, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077852, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077852, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077726, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077726, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077790, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077790, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077768, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077768, ip=10.120.204.29)[0m   warnings.warn(
[2m[36m(iter_task pid=3077858, ip=10.120.204.29)[0m /home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/dask/config.py:693: UserWarning: Configuration key "shuffle" has been deprecated. Please use "dataframe.shuffle.algorithm" instead
[2m[36m(iter_task pid=3077858, ip=10.120.204.29)[0m   warnings.warn(
Iter [4]
Iter [6]
Iter [8]
Iter [10]
Iter [12]
Iter [14]
Iter [16]
Iter [18]
The node with node id: 0faa039eb28fe07dc61f089565dffe70449056be86aeddccfa0f7b2f and address: 172.30.204.33 and node name: 172.30.204.33 has been marked dead because the detector has missed too many heartbeats from it. This can happen when a 	(1) raylet crashes unexpectedly (OOM, preempted node, etc.) 
	(2) raylet has lagging heartbeats due to slow network or busy workload.
EST_ANALYTICS_TIME:  200.13015 (avg:10.00651)
Traceback (most recent call last):
  File "/mnt/lustre/scratch/nlsas/home/ulc/cursos/curso341/dask_on_reisa_tests/E4/DASK_ON_REISA/reduction/P4-SN2-LS536-GS2147-I20-AN2-D2025-03-10_08-46-49/reduction.py", line 56, in <module>
    result = handler.get_result(process_func, iter_func, selected_iters=iterations, kept_iters=max_iterations,
  File "/mnt/lustre/scratch/nlsas/home/ulc/cursos/curso341/dask_on_reisa_tests/E4/DASK_ON_REISA/reduction/P4-SN2-LS536-GS2147-I20-AN2-D2025-03-10_08-46-49/reisa.py", line 127, in get_result
    tmp = ray.get(results)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/_private/client_mode_hook.py", line 104, in wrapper
    return getattr(ray, func.__name__)(*args, **kwargs)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/util/client/api.py", line 42, in get
    return self.worker.get(vals, timeout=timeout)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/util/client/worker.py", line 434, in get
    res = self._get(to_get, op_timeout)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/util/client/worker.py", line 462, in _get
    raise err
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/util/client/server/server.py", line 489, in _get_object
    items = ray.get(objectrefs, timeout=request.timeout)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/_private/client_mode_hook.py", line 105, in wrapper
    return func(*args, **kwargs)
  File "/home/ulc/cursos/curso341/.local/lib/python3.9/site-packages/ray/_private/worker.py", line 2521, in get
    raise value.as_instanceof_cause()
ray.exceptions.RayTaskError: [36mray::iter_task()[39m (pid=3077453, ip=172.30.204.29)
  File "/mnt/lustre/scratch/nlsas/home/ulc/cursos/curso341/dask_on_reisa_tests/E4/DASK_ON_REISA/reduction/P4-SN2-LS536-GS2147-I20-AN2-D2025-03-10_08-46-49/reisa.py", line 116, in iter_task
    current_results_list = ray.get(current_results_list)  # type: #List[dask.array.core.Array]
ray.exceptions.OwnerDiedError: Failed to retrieve object c4a1b107e7175e6fffffffffffffffffffffffff0100000001000000. To see information about where this ObjectRef was created in Python, set the environment variable RAY_record_ref_creation_sites=1 during `ray start` and `ray.init()`.

The object's owner has exited. This is the Python worker that first created the ObjectRef via `.remote()` or `ray.put()`. Check cluster logs (`/tmp/ray/session_latest/logs/*9a01f2ba63492f3bb24517b6bc7159f4ad2a6dc8c044bcfdb88f45d0*` at IP address 172.30.204.33) for more information about the Python worker failure.
srun: error: ilk-173: task 0: Exited with exit code 1
srun: launch/slurm: _step_signal: Terminating StepId=10927293.6
srun: Job step aborted: Waiting up to 32 seconds for job step to finish.
slurmstepd: error: *** JOB 10927293 ON ilk-172 CANCELLED AT 2025-03-10T12:34:00 DUE TO TIME LIMIT ***
slurmstepd: error: *** STEP 10927293.7 ON ilk-177 CANCELLED AT 2025-03-10T12:34:00 DUE TO TIME LIMIT ***
*** SIGTERM received at time=1741606440 on cpu 18 ***
*** SIGTERM received at time=1741606440 on cpu 52 ***
*** SIGTERM received at time=1741606440 on cpu 0 ***
[failure_signal_handler.cc : 329] RAW: Signal 15 raised at PC=0x14834bac754c while already in AbslFailureSignalHandler()
*** SIGTERM received at time=1741606440 on cpu 18 ***
[failure_signal_handler.cc : 329] RAW: Signal 15 raised at PC=0x151e4dcab54c while already in AbslFailureSignalHandler()
*** SIGTERM received at time=1741606440 on cpu 52 ***
